src/steps/GithubActionStep.ts

```diff
@@ -0,0 +1,25 @@
+import { Step } from 'aws-cdk-lib/pipelines';
```

Comment:
can we rename this file `github-action-step.ts`

Reply:
updated.